src/test/java/org/jenkinsci/plugins/workflow/job/CpsPersistenceTest.java

```diff
@@ -197,10 +198,10 @@ public void completedFinalFlowNodeNotPersisted() throws Exception {
             String finalId = run.getExecution().getCurrentHeads().get(0).getId();
 
             // Hack but deletes the file from disk
-            CpsFlowExecution cpsExec = (CpsFlowExecution)(run.getExecution());
+            CpsFlowExecution cpsExec = (CpsFlowExecution) run.getExecution();
             File f = new File(cpsExec.getStorageDir(), finalId+".xml");
-            f.delete();
-            executionAndBuildResult[0] = ((CpsFlowExecution)(run.getExecution())).getResult();
+            Files.delete(f.toPath());
```

Comment:
This will throw an exception if the file doesn't exist where the old code wouldn't have, but I think that is a good thing, since it keeps us from unintentionally testing a different scenario or potentially having a test pass for the wrong reason.